packages/web/src/components/stat-banner/StatBanner.tsx

```diff
@@ -192,6 +193,7 @@ export const StatBanner = (props: StatsBannerProps) => {
   const followButtonRef = useRef<HTMLButtonElement>(null)
   const isManagedAccount = useIsManagedAccount()
   const chatPermissionStatus = useSelector(getChatPermissionsStatus)
+  const { data: currentUserId } = useGetCurrentUserId({})
```

Comment:
is this using a selector or does it make an actual API call? I think in the past we've used `useSelector(getUserId)` or similar for stuff like this

Reply:
this calls `const account = await audiusBackend.getAccount()` which means we rely on `userStateManager` so there shouldn't be a network call

Reply:
we should be moving away from audiusBackend generally imo, probably preferred to use a selector here? @schottra what do you think

Reply:
this is an audius-query call that we use in ~100 call sites currently. We'll be replacing it when we move to tanstack query anyway

Reply:
yeah not blocking. I'm just curious since this doesn't seem like it needs to do any async work but rather just check state

Reply:
yeah that's correct, and it gets cached in audius-query's state, so all but the first call will be just selecting state from the audius query slice anyway. It's def duplicated work at the moment, but more similar in usage to our expected long-term patterns 🤷

Reply:
I believe `getAccount` is `async` because the first time needs to wait for libs to finish initializing.
However, I have a PR in progress that revises a bunch of this stuff. We can continue to use `useGetCurrentUserId` here. It will get migrated with the rest of the instances.